halo/evmengine/keeper/abci_internal_test.go

```diff
@@ -332,7 +338,9 @@ func newMockEngineAPI() (mockEngineAPI, error) {
 	}, nil
 }
 
-type mockAddressProvider struct{}
+type mockAddressProvider struct {
```

Comment:
can also do `type mockAddressProvider string` since it only ever does one thing.